org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/rule/RuleDescriptionPanel.java

```diff
@@ -109,10 +108,6 @@ public void updateRule(Either<RuleMonolithicDescriptionDto, RuleSplitDescription
               contextualTabFolder.setSelection(contextualTabItem);
             }
 
-            var contextualTabTitle = new StyledText(contextualTabFolder, SWT.NONE);
```

Comment:
How are the contextual tab titles handled now? Shouldn't them be replaced by a normal `Label`?

Reply:
The tab title is set on the `TabItem`, few line before